resources/views/pages-legacy/userInfo.blade.php

```diff
@@ -54,6 +56,23 @@
 $userCompletedGamesList = getUsersCompletedGamesAndMax($userPage);
 $userAwards = getUsersSiteAwards($userPageModel);
 
+// Identify subset game IDs. We'll exclude these from the user's average completion %.
+$gameIds = collect($userCompletedGamesList)->pluck('GameID')->filter()->unique();
+$subsetGameIds = [];
+if ($gameIds->isNotEmpty()) {
+    $subsetGameIds = GameAchievementSet::whereIn('game_id', $gameIds)
+        ->where('type', AchievementSetType::Core)
+        ->whereExists(function($query) {
+            $query->select('*')
+                ->from('game_achievement_sets as gas2')
+                ->whereColumn('gas2.achievement_set_id', 'game_achievement_sets.achievement_set_id')
+                ->whereColumn('gas2.game_id', '!=', 'game_achievement_sets.game_id')
+                ->where('gas2.type', '!=', AchievementSetType::Core);
+        })
+        ->pluck('game_id')
+        ->toArray();
+}
```

Comment:
Is this really necessary? Can't we just exclude any game_id that is used as a non-core set?
```
    $subsetGameIds = GameAchievementSet::whereIn('game_id', $gameIds)
        ->where('type', '!=', AchievementSetType::Core)
        ->pluck('game_id')
        ->toArray();
```

Reply:
Good point - this often trips me up. Updated in latest.